libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/arrow/api/MapboxRouteArrowView.kt

```diff
@@ -33,7 +35,11 @@ import com.mapbox.navigation.utils.internal.logE
  * @param options the options used for determining the rendering appearance and/or behavior.
```

Comment:
```suggestion
 * If you're recreating the `MapboxRouteArrowView` instance, for example to change the [options], make sure that your first interaction restores the state and re-applies the options by calling `MapboxRouteLineApi#redraw` and passing it on to `MapboxRouteLineView#render`. 
 *
 * @param options the options used for determining the rendering appearance and/or behavior.
```

Reply:
My interpretation of what you've written is that if there's an arrow on the map, following these instructions will redraw the arrow.  They won't. The arrow is driven by route progress. Redrawing the route line has no effect on the arrows.  What was your intended instruction here?